Implement keep filter to preserve matching images (#6)
danielfm/kube-ecr-cleanup-controller: README.md

    	just log, don't delete any images.
  -interval int
    	check interval, in minutes. (default 30)
  -keep-filters string

>         specify a filter or regex that when matched will preserve the matching images.

Reviewer: The description here doesn't match the code.

```suggestion
        comma-separated list of filters or regexes that when matched will preserve the matching images.
```
